phpcs.xml

```diff
@@ -31,6 +31,7 @@
     <exclude name="NeutronStandard.StrictTypes.RequireStrictTypes.StrictTypes" />
     <exclude name="Squiz.Commenting.FileComment.Missing" />
     <exclude name="Squiz.Commenting.FileComment.MissingPackageTag" />
+    <exclude name="Generic.Arrays.DisallowShortArraySyntax.Found" />
```

Comment:
I've added to this to other plugins of ours and am just getting around to adding it here 😄 There is nothing wrong with short array syntax!

`array()` vs `[]`